engine/src/test/java/org/camunda/bpm/engine/test/dmn/deployment/DecisionDefinitionTest.java

```diff
@@ -72,10 +75,20 @@ public class DecisionDefinitionTest {
   protected HistoryService historyService;
 
   @Before
-  public void init() {
+  public void init() throws ParseException {
     this.repositoryService = engineRule.getRepositoryService();
     this.decisionService = engineRule.getDecisionService();
     this.historyService = engineRule.getHistoryService();
+
+    SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy hh:mm:ss.SSS");
```

Comment:
**Context:** Fixes Flaky DST issues with this test
**Why:** The change is not related to this PR but will be committed together since it was identified